MFVNeutralino/test/TriggerStudies/draw_eff.py

```diff
@@ -15,26 +20,27 @@
 
 which = typed_from_argv(int, 0)
 data_period, int_lumi = [
-    ('p8',101037.),
-    ('',   41525.),
-    ('B',   4794.),
-    ('C',   9631.),
+    ('p8',100293.),
+    ('',   40610.),
+    ('B',   4803.),
+    ('C',   9573.),
     ('D',   4248.),
-    ('E',   9314.),
-    ('F',  13538.),
-    ('',   59512.),
-    ('A',  14002.),
-    ('B',   7091.),
-    ('C',   6937.),
-    ('D',  31482.),
+    ('E',   9315.),
+    ('F',  12671.),
+    ('',   59683.),
+    ('A',  13978.),
+    ('B',   7064.),
+    ('C',   6895.),
+    ('D',  31747.),
```

Comment:
Are these updated lumi values the ones corresponding to the e/mu triggers? The difference is probably small / unimportant, but wanted to check.

Reply:
Oh that's a nice point. I used the lumi in ana_*.json files so actually that is luminosity for METNoMu trigger. Maybe I should use the corresponding mu/ele trigger lumis for that.

Reply:
Also, we should make all of this script configurable to work with MET or HT or other triggers.

Reply:
I think for other triggers, the important things to change is just the directory of root files and plots, the details such as name and ranges for the plot and `ns` variable which controls the name of the plot for efficiency calculation. Those variables are easy to change and I think I cannot do that now because I don't really know those details about other triggers. Please let me know if just keeping what it is now is ok or you have something specific in mind that I can change. Thanks!